app/presenters/track.rb

```diff
@@ -17,21 +17,21 @@ def method_missing(method, *args)
       def docs
         track_docs = trackler_track.docs("/api/v1/tracks/%s/images/docs/img" % @track_id)
 
-        track_docs.each_pair do |topic_name, topic_content|
-          track_docs[topic_name] = if topic_content.present?
-                                     topic_content
-                                   else
-                                     fallback_topic_content(topic_name)
-                                   end
+        track_docs.each_pair do |topic_key, topic_content|
```

Comment:
Note for next time: It would have been nice to make this name change a separate commit.